### ISSUE TITLE
Primo - JS files/Accessibility Menu - plugin

### ISSUE THREAD

**Issue by antrez770** [ExLibrisGroup/primo-explore-devenv#65]
Hi, What is the right way to add custom JS - files to Primo? 
Is somebody use Accessibility menu / plugin for Primo?
Thank You.

**Comment by danmichaelo** [comment]
For general questions, please try the #primodev channel: https://igelu-eluna-siwg.slack.com/channels/primodev

Btw. a link to that channel should be added to the README?

**Comment by antrez770** [comment]
Hi Now I add to this group, but looking for method How call jQuery function in AngularJS controller?

**Comment by cooldudezach** [comment]
@antrez770 I copied the contents of the jQuery.js file to the very beginning of custom.js

**Comment by antrez770** [comment]
@cooldudezach and it is working? can You give me any example?

**Comment by cooldudezach** [comment]
@antrez770 Here is the production copy of my custom.js [https://csudh-primo.hosted.exlibrisgroup.com/primo-explore/custom/01CALS_UDH/js/custom.js](url)

**Comment by antrez770** [comment]
I can't access, can You upload it anywhere?

**Comment by antrez770** [comment]
@cooldudezach can You please write link to Your Primo, the link You gave me is not uploaded.

**Comment by cooldudezach** [comment]
@antrez770 I fixed the link. Do a search in the file for both `$(` and `jQuery(`

**Comment by antrez770** [comment]
@cooldudezach Can You do with Jquery in custom.js something like in my previous example? Or somethink like any change (color, background) by click on link?

**Comment by cooldudezach** [comment]
@antrez770 If you go through the code in the link I posted, there are a couple examples. In fact, our mobile menu is run on jquery.

Try this code: `$('<div id="tools" style="text-align:right;position: absolute;bottom: 0px;"><div id="mylink">Click me</a></div>').insertAfter('body'); $("#mylink").click(function() { $('body').css({'background-color':'red'}); });` (I changed some of the css, it works for me)

**Comment by antrez770** [comment]
Yes it work thank You. Do You know can I insert this div into body? Is it possible?

**Comment by antrez770** [comment]
Hi @cooldudezach the solution I insert some changes but it is work. but there is one missing step, I can't reach the <div> with tab focus also if it has tabindex="1" do You know any solution in this specific case?
Thank You.

**Comment by antrez770** [comment]
Sorry I solved it

**Comment by Maluzzz** [comment]
Hi! How can I join the slack channel :D

**Comment by gilax** [comment]
hi @antrez770, can i close this issue?
